nncf/experimental/onnx/graph/onnx_graph.py

```diff
@@ -117,6 +117,39 @@ def get_node_edges(self, node_name: str) -> Dict[str, List[ValueInfoProto]]:
                         'output': list(node.output)}
         raise RuntimeError('There is no node with the name {}'.format(node_name))
 
+    def get_input_port_id_for_nodes_after_input(self, input_name: str, to_node: NodeProto):
```

Comment:
Consider redesigning these functions in the same manner as described in the comment above *if* the `get` result not being found is a regular thing. Also, add return type hints.

Reply:
The logic of this function the same as above